analytics.js

```diff
@@ -174,6 +176,23 @@ class Analytics {
         this.loadOnlyIntegrations,
         this.clientIntegrations
       );
+      // Check if cookie consent manager is being set through load options
+      if (this.cookieConsentOptions) {
+        // Call the cookie consent factory to initialise and return the type of cookie
+        // consent being set. For now we only support OneTrust.
+        this.cookieConsent = CookieConsentFactory.initialize(
```

Comment:
This doesn't have to be class-level variable

Reply:
Please correct this on the staging branch as well

Reply:
done